<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QMainWindow" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainInterface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainInterface{
border-image: url(src/res/darkgray.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget{
border-image: url(src/res/darkgray.png);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pb_open">
         <property name="styleSheet">
          <string notr="true">image: url(src/res/open.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_save">
         <property name="styleSheet">
          <string notr="true">image: url(src/res/save.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_spin">
         <property name="styleSheet">
          <string notr="true">image: url(src/res/edit.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_curve">
         <property name="styleSheet">
          <string notr="true">image: url(src/res/color.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_text">
         <property name="styleSheet">
          <string notr="true">image: url(src/res/other.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_beauty">
         <property name="styleSheet">
          <string notr="true">image: url(src/res/beauty.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">#widget_2{
border-image: url(src/res/darkgray.png);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_show">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>950</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(223, 223, 223);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="open"/>
    <addaction name="save"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="ac_crop"/>
    <addaction name="ac_spin"/>
    <addaction name="ac_exp"/>
    <addaction name="ac_bri"/>
    <addaction name="ac_conra"/>
    <addaction name="ac_hisequ"/>
    <addaction name="ac_sha"/>
    <addaction name="ac_smo"/>
    <addaction name="ac_coltem"/>
    <addaction name="ac_tone"/>
   </widget>
   <widget class="QMenu" name="menu_color">
    <property name="title">
     <string>色彩(&amp;C)</string>
    </property>
    <addaction name="ac_lis"/>
    <addaction name="ac_curadj"/>
    <addaction name="ac_satadj"/>
    <addaction name="ac_curtone"/>
    <addaction name="ac_hsl"/>
   </widget>
   <widget class="QMenu" name="menu_others">
    <property name="title">
     <string>其它(&amp;W)</string>
    </property>
    <addaction name="ac_text"/>
    <addaction name="ac_clearfog"/>
    <addaction name="ac_watermark"/>
    <addaction name="ac_beauty"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_color"/>
   <addaction name="menu_others"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="ac_crop">
   <property name="text">
    <string>构图</string>
   </property>
  </action>
  <action name="ac_spin">
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
  <action name="ac_exp">
   <property name="text">
    <string>曝光</string>
   </property>
  </action>
  <action name="ac_bri">
   <property name="text">
    <string>亮度</string>
   </property>
  </action>
  <action name="ac_conra">
   <property name="text">
    <string>对比度</string>
   </property>
  </action>
  <action name="ac_sha">
   <property name="text">
    <string>锐化</string>
   </property>
  </action>
  <action name="ac_smo">
   <property name="text">
    <string>平滑</string>
   </property>
  </action>
  <action name="ac_lis">
   <property name="text">
    <string>光感</string>
   </property>
  </action>
  <action name="ac_curadj">
   <property name="text">
    <string>曲线调整</string>
   </property>
  </action>
  <action name="ac_satadj">
   <property name="text">
    <string>饱和度调整</string>
   </property>
  </action>
  <action name="ac_hsl">
   <property name="text">
    <string>HSL</string>
   </property>
  </action>
  <action name="ac_coltem">
   <property name="text">
    <string>色温</string>
   </property>
  </action>
  <action name="ac_tone">
   <property name="text">
    <string>色调</string>
   </property>
  </action>
  <action name="ac_text">
   <property name="text">
    <string>文字</string>
   </property>
  </action>
  <action name="ac_clearfog">
   <property name="text">
    <string>去雾</string>
   </property>
  </action>
  <action name="ac_watermark">
   <property name="text">
    <string>水印</string>
   </property>
  </action>
  <action name="ac_beauty">
   <property name="text">
    <string>美颜</string>
   </property>
  </action>
  <action name="ac_hisequ">
   <property name="text">
    <string>直方图均衡化</string>
   </property>
  </action>
  <action name="ac_curtone">
   <property name="text">
    <string>曲线调色</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
